pdfminer/pdffont.py

```diff
@@ -508,13 +508,21 @@ def __init__(self, descriptor, widths, default_width=None):
         self.hscale = self.vscale = .001
         self.glyph_bounding_box = None
         self.font_bounding_box = None
+        self.default_ascent = 100
```

Comment:
Can you comment what these numbers are? They seem quite large for font size/pixels/etc. Is there a special pdf scale?

Reply:
Added a comment above regarding units. They are "glyph space" units.

Reply:
Thanks, this is great comment